eZ/Publish/Core/Repository/UserService.php

```diff
@@ -739,13 +740,13 @@ public function updateUser(APIUser $user, UserUpdateStruct $userUpdateStruct): A
     /**
      * Validates and updates just the user's password.
      *
-     * @param \eZ\Publish\API\Repository\Values\User\User $user
-     * @param string $newPassword
+     * @return \eZ\Publish\API\Repository\Values\User\User
```

Comment:
nitpick: Unnecessary `@return` declaration - it matches the method's return type afaik.